wolox-books/src/app/screens/list-books/screens/book-detail/book-detail.component.html

```diff
@@ -1 +1,22 @@
-<p>book-detail works!</p>
+<div class="content">
+  <a class="back" href="/books">Atás</a>
+  <section class="content-detail">
+    <div class="section-img">
+      <img class="img-book" alt="img-boos" [src]="details.image_url" />
+    </div>
+    <div class="section-info">
+      <div class="content-title">
+        <h3 class="text-title">{{ details.title }}</h3><span class="text-genre">({{ details.genre }})</span>
```

Comment:
This can't be h3, you could use span, but remember, to use a h3 you should've used h1 and h2